### ISSUE TITLE
y_hat and y_i_hat

### ISSUE THREAD

**Issue by pfistfl** [compstat-lmu/latex-math#7]
```
% yhat
\newcommand{\yh}{\hat{y}}                                                   % y hat for prediction of target
\newcommand{\yih}{\hat{y}}                                                  % y hat for prediction of target
```

should this be:  `\newcommand{\yih}{\hat{\yi}}                                                  % y hat for prediction of target`?

**Comment by pfistfl** [comment]
@schalkdaniel

**Comment by schalkdaniel** [comment]
I know it's a very old issue, but it should be resolved by now (@berndbischl was the one 9b0f0c1f):
https://github.com/compstat-lmu/latex-math/blob/52222b1cf70d240b5b11e679ce0604add184d124/basic-ml.tex#L72
